ontology/ogi.md

```diff
@@ -3,11 +3,16 @@ layout: ontology_detail
 id: ogi
 description: An ontology that formalizes the genomic element by defining an upper class genetic interval
 homepage: https://code.google.com/p/ontology-for-genetic-interval/
+contact:
+  label: Asiyah Yu Lin
+  email: linikujp@gmail.com
+  github: linikujp
+  orcid: 0000-0002-5379-5359
 products:
   - id: ogi.owl
 title: Ontology for genetic interval
 tracker: https://code.google.com/p/ontology-for-genetic-interval/issues/list
-activity_status: orphaned
+activity_status: inactive
```

Comment:
change to "obsolete" and add a replaced_by link to ogsf (as per email discussions with @linikujp)

Keep contact information the way you have it here!

Reply:
done, but obsolete is its own field unrelated to activity status so note that it is now marked as inactive